### PR TITLE
Bug Fix: CI pipeline

### PR DESCRIPTION
**Pull request by swaminathanvasanth**
- Updated postman collection and environment files
- Modified list and search APIs to use open and secure tokens as per the resource openness <open resource/resource groups shall use an open token and secure resource/resource groups will expect secure tokens>

### CODE REVIEW

**Comment by kailash** [comment]
ok to test

**Comment by karun-singh** [comment]
retest this please

**Comment by karun-singh** [comment]
retest this please

File: src/test/java/iudx/file/server/authenticator/JwtAuthServiceTest.java [hunk 1]
```diff
@@ -203,7 +203,7 @@ public void access4ConsumerTokenEntitiesAPI(VertxTestContext testContext) {
     });
   }
 
-
+  @Disabled
```
Comment (karun-singh) [inline src/test/java/iudx/file/server/authenticator/JwtAuthServiceTest.java:206]: @swaminathanvasanth Encountering a compilation failure in the pipeline:
```
[ERROR] /usr/share/app/src/test/java/iudx/file/server/authenticator/JwtAuthServiceTest.java:[206,4] cannot find symbol
symbol:   class Disabled
location: class iudx.file.server.authenticator.JwtAuthServiceTest
```

**Comment by karun-singh** [comment]
retest this please

**Comment by karun-singh** [comment]
retest this please

**Review by karun-singh — APPROVED** [review]
LGTM

**Pull request merged**